handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/filterext/IsolationDiscoveryFilter.java

```diff
@@ -44,7 +44,7 @@ public class IsolationDiscoveryFilter implements ServerListFilterExt {
   private static final Logger LOGGER = LoggerFactory.getLogger(IsolationDiscoveryFilter.class);
 
   private final DynamicBooleanProperty emptyProtection = DynamicPropertyFactory.getInstance()
-      .getBooleanProperty(EMPTY_INSTANCE_PROTECTION, false);
+      .getBooleanProperty(EMPTY_INSTANCE_PROTECTION, true);
```

Comment:
does this is a default value change? should we put it in the upgrade guide?

Reply:
I changed back